include/rootston/seat.h

```diff
@@ -9,6 +9,9 @@ struct roots_drag_icon {
 	struct wl_list link; // roots_seat::drag_icons
 	bool mapped;
 
+	bool is_pointer;
+	int32_t touch_id;
```

Comment:
This should be called slot for consistency.

Reply:
I don't think consistency is possible here. Libinput gives us "slots" but the seats work with "ids" (named that in the wl-touch interface) which can be slots or anything else depending on what the backend names it. The point at which the terminology changes I'm not sure, but rootston does work with multiple backends.

It's a little sloppy for sure, so I'll give you guys the final decision now that you know that.

Reply:
Alright, IDs is fine if that's the wl terminology.

Reply:
I think the wlroots touch event struct should use `id` too then, since it shouldn't be libinput-dependent.

Reply:
The way to be consistent would be to call them "slots" all the way through (breaking consistency with the wayland interface names though) or change the event struct to call it a touch_id so it is consistent all throughout rootston.

Reply:
>change the event struct to call it a touch_id so it is consistent all throughout rootston.

That seems better.